wordpress-mobile/WordPress-iOS
# Plans: Fix truncated feature description labels.

Fixes #10310 

The issue was due to cell heights fixed to 80 px.  Any wrapping text was hidden because the cells were too small.  This PR tweaks constraints and row sizing in the plans storyboard and specifies `.automaticDimension` for row height.

To test:
On a wpcom blog:
- Go to My Sites > Site > Plans and tap to view a plan detail screen
- Confirm that the feature descriptions wrap their text and are not truncated by a cell that is not high enough.
- Check the console and confirm there are no breaking constraints due to the change.

![simulator screen shot - iphone 8 - 2018-10-23 at 16 57 41](https://user-images.githubusercontent.com/1435271/47394003-e5bd2300-d6e6-11e8-851d-a4a667cbd83f.png)

@ScoutHarris could I trouble you with this one?
